### ISSUE TITLE
opretter gentaget event, for 2 events og forsvinder derfor helt fra siden. 

### ISSUE THREAD

**Issue by Siggik** [ronan-mch/det_sker#392]
List svært at forklarer. Men tænker at vi har et ux problem når disse 2 fine event https://www.dukop.dk/events/1971?locale=da

Blir oprettet således at de først dukker op på for almindelige brugere af siden i start januar.

**Comment by ronan-mch** [comment]
Ja, det kan jeg godt se, måske mangler der en forklarende test et eller
andet sted på formularen?

**Comment by Siggik** [comment]
jo, jeg ser lige på det en af dagene. Nu har jeg problemet midlertidigt, ved at oprette eventsene på den rigtige måde for dem.

**Comment by Siggik** [comment]
Måske skal der med lille blåskrift stå. Eller måske et lille spørgsmålsteng hvor der popper tekst op? Hvordan tror du det vil virke bedste: "Hvis du oprettet dit event som et gentagende event, optræder eventet til højre på forsiden under "Faste begivenhder" fra den uge du starter begivenhedsrækken. Brug denne funktion til begivenheder der gentager sig selv mange gange over flere måneder. Hvis den kun gentager sig 2-5 gange, så opret hellere begivenhederne enkelvis."

**Comment by benjaoming** [comment]
@larsgaardsoe den her er ret interessant ift. Faste Begivenheder. Tidligere er bare blevet vist indeværende uges Faste Begivenheder. Men der burde være en mulighed for at gå til næste uge også, eller få vist en fuld oversigt / kalendervisning inkl. faste begivenheder.
